tests/integration/helm_default_installation_test.go

```diff
@@ -268,5 +269,56 @@ func Test_Helm_Default_FluentD_Metadata(t *testing.T) {
 		).
 		Feature()
 
-	testenv.Test(t, featInstall, featMetrics)
+	featLogs := features.New("logs").
+		Setup(stepfuncs.GenerateLogsWithDeployment(
+			logsGeneratorCount,
+			internal.LogsGeneratorName,
+			internal.LogsGeneratorNamespace,
+			internal.LogsGeneratorImage,
+		)).
+		Assess("logs from log generator present", stepfuncs.WaitUntilExpectedLogsPresent(
+			logsGeneratorCount,
+			map[string]string{
+				"namespace":      internal.LogsGeneratorName,
+				"pod_labels_app": internal.LogsGeneratorName,
+			},
+			internal.ReceiverMockNamespace,
+			internal.ReceiverMockServiceName,
+			internal.ReceiverMockServicePort,
+			waitDuration,
+			tickDuration,
+		)).
+		Assess("expected container log metadata is present", stepfuncs.WaitUntilExpectedLogsPresent(
+			logsGeneratorCount,
+			map[string]string{
+				"namespace":      internal.LogsGeneratorName,
+				"pod_labels_app": internal.LogsGeneratorName,
+				"container":      internal.LogsGeneratorName,
+				"deployment":     internal.LogsGeneratorName,
+				"replicaset":     "",
```

Comment:
Just a song of the future: these could be made a bit more sophisticated so that the function itself accepts not `map[string]string` but an interface that instead of matching only 1:1 could also be implemented as regex matcher.

Reply:
Could they? We're restricted by what receiver-mock allows, and what we give it are just strings.

Reply:
You could get creating and limit search by one set of rules and then match via regex the results.

Or implement regex matching in receiver-mock 🚀